lib/Notification/Notifier.php

```diff
@@ -287,6 +293,62 @@ protected function shortenJsonEncodedMultibyteSave(string $subject, int $dataLen
 		return $temp;
 	}
 
+	private function parseStoredRecording(
+		INotification $notification,
+		Room $room,
+		Participant $participant,
+		IL10N $l
+	): INotification {
+		$parameters = $notification->getSubjectParameters();
+		$shareAction = $notification->createAction()
+			->setParsedLabel($l->t('Share to chat'))
+			->setPrimary(true)
+			->setLink(
+				$this->urlGenerator->linkToRouteAbsolute(
+					'ocs.spreed.Chat.shareObjectToChat',
+					[
+						'apiVersion' => 'v1',
+						'objectType' => 'file',
+						'objectId' => $notification->getObjectId(),
```

Comment:
```suggestion
						'objectId' => $parameters['objectId'],
```

Otherwise you add the room token instead of the file Id as a share